scripts/include_list.php

```diff
@@ -19,8 +19,8 @@
 <div class="customlabels2 column1">
 <form action="" method="GET" id="add">
   <h2>All Species Labels</h2>
-  <select name="species[]" id="species" multiple size="30">
-    <option selected value="base">Please Select</option>
+  <input autocomplete="off" size="18" type="text" placeholder="Search Species..." id="species_searchterm" name="species_searchterm">
+  <select name="species[]" id="species" multiple size="30">ol
```

Comment:
size="30" makes it kind of tricky to scroll beyond the selectbox as it occupies the whole screen (on my phone at least)
Could you keep the size at 25?

also: what is 'ol' ?

Reply:
thanks! changed to 25. I guess "ol" was a little typo ;-)

Reply:
changed also for the other table